docs/rfcs/0000-template.md

```diff
@@ -0,0 +1,82 @@
+- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
```

Comment:
```suggestion
# Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
```

Reply:
Nicer heading levels are better for screen readers, in addition to being more standard.